incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -256,6 +276,9 @@ func CheckHNCPath() {
 }
 
 func RecoverHNC() {
+	if hncRecoverPath == "" {
+		return
+	}
```

Comment:
I thought it already existed somewhere?

Reply:
In `BeforeEach` we call `Skip()` if this var isn't set, but I _believe_ based on what I saw that `AfterEach` is still called anyway, which makes sense because there might be things we want to clean up. I _think_ we haven't noticed this before because we ~always have `HNC_REPAIR` set when running tests (at least I do).

What my tests showed is that all tests _before_ the skipped tests worked fine, and all tests that ran _after_ the skipped tests failed because the service was broken. When I looked, the HNC deployment had been deleted, which is all this function will do if `HNC_REPAIR` is missing. Adding this line fixed the problem.

Reply:
Thanks for the explanation! It would be nice to have these as a comment too.

Reply:
Good point! Done